shardingsphere-proxy/shardingsphere-proxy-bootstrap/src/main/resources/conf/server.yaml

```diff
@@ -81,3 +81,4 @@
 #  sql-federation-enabled: false
 #    # Available proxy backend driver type: JDBC (default), ExperimentalVertx
 #  proxy-backend-driver-type: JDBC
+#  data-source-aggregation-enabled: false
```

Comment:
Please keep a blank line in the end of line.

Reply:
Fix in https://github.com/apache/shardingsphere/pull/16710